### ISSUE TITLE
Tray-Application

### ISSUE THREAD

**Issue by tiptronic** [patr0nus/DeskGap#27]
Thanks for this nifty project (there was an attempt a while ago by the apache foundation building such a thing using Cordova [https://cordova.apache.org](https://cordova.apache.org) ) but that stalled.
Your approach is pretty good and works really well.
Since the application size comes way down, it seems as the project is also pretty good predestined to create tray/menubar-extra applications. But as far as I can see, there's no support for that - is that right?

**Comment by mdings** [comment]
You can use [systray](https://www.npmjs.com/package/systray) in the main process. I just tried it out and seems to be working as expected

**Comment by tiptronic** [comment]
Yes, I know I can do that (and I did), but it would be a great addition to this nice (but dead?) project.
